app/src/main/kotlin/com/wire/android/ui/userprofile/image/AvatarPickerViewModel.kt

```diff
@@ -31,9 +32,10 @@ class AvatarPickerViewModel @Inject constructor(
     private val dataStore: UserDataStore,
     private val getUserAvatar: GetPublicAssetUseCase,
     private val uploadUserAvatar: UploadUserAvatarUseCase,
+    private val avatarImageManager: AvatarImageManager
 ) : ViewModel() {
 
-    var avatarRaw by mutableStateOf<ByteArray?>(null)
+    var pictureState by mutableStateOf<PictureState>(PictureState.Initial("".toUri()))
```

Comment:
I think there is a Uri.Empty() os somethinmg like that

Reply:
maybe it will make sense to introduce another object like PictureState.Empty() or asomething like that, what do you think ?

Reply:
We can't, sadly as `Uri` is an android class, mocking EMPTY it's not possible fot testing
But anyway, I added an Empty state with the same `toUri()` call 👍